Gradle build is broken when using detox

I agree to the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Using [detox](https://github.com/wix/Detox) on android with react-native-auth0 breaks the gradle build.

### Reproduction

- Install detox and [setup android](https://github.com/wix/Detox/blob/master/docs/Introduction.Android.md)
- Install react-native-auth0 and configure
- Attempt to run `cd android && ./gradlew :app:assembleRelease assembleAndroidTest -DtestBuildType=release`

Log:
```
> Transform artifact browser-1.2.0.aar (androidx.browser:browser:1.2.0) with DexingNoClasspathTransform
AGPBI: {"kind":"error","text":"Static interface methods are only supported starting with Android N (--min-api 24): androidx.browser.trusted.TrustedWebActivityDisplayMode androidx.browser.trusted.TrustedWebActivityDisplayMode.fromBundle(android.os.Bundle)","sources":[{}],"tool":"D8"}

> Task :react-native-auth0:mergeExtDexDebugAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-auth0:mergeExtDexDebugAndroidTest'.
> Could not resolve all files for configuration ':react-native-auth0:debugAndroidTestRuntimeClasspath'.
   > Failed to transform browser-1.2.0.aar (androidx.browser:browser:1.2.0) to match attributes {artifactType=android-dex, dexing-enable-desugaring=false, dexing-is-debuggable=true, dexing-min-sdk=18, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingNoClasspathTransform: /Users/ian/.gradle/caches/transforms-2/files-2.1/929c1ed6faf5ef6e7530d107e8be13eb/browser-1.2.0-runtime.jar.
         > Error while dexing.
```

Please see [this issue](https://github.com/FormidableLabs/react-native-app-auth/issues/455) for more information

### Environment

- **Version of this library used:** 2.5.0
- **Version of the platform or framework used, if applicable:** react-native 0.63.1
- **Other relevant versions (language, server software, OS, browser):** typescript, Mac OS 10.15.5
- **Other modules/plugins/libraries that might be involved:** [detox](https://github.com/wix/Detox)

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -37,6 +37,10 @@ android {
   lintOptions {
     abortOnError false
   }
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
 }
 
 repositories {

